### ISSUE TITLE
Lost panels

### ISSUE THREAD

**Issue by ennuished** [adenzu/Manga-Panel-Extractor#5]
Hello,The issue i'm facing is that all the panels are not being extracted and some just disappear.

https://github.com/adenzu/Manga-Panel-Extractor/assets/166026888/994d070b-03df-44f6-b123-4bbcd0eaa3dd

**Comment by adenzu** [comment]
Hi, thanks for your feedback! However I'm afraid the issue you are facing is not a bug or unintended behavior. It's just that the program is not good enough.  There are manga pages that face the same issue too (see [here](https://github.com/adenzu/Manga-Panel-Extractor/blob/main/INSIGHT.md)).

There is also the fact that I have been developing this project with manga style in mind. I actually had mentioned manhwa and other media types in README.md under [Usage](https://github.com/adenzu/Manga-Panel-Extractor?tab=readme-ov-file#executable) section (under Executable, below last step). If I were to include manhwa in the valid media types then the whole project is better off as converted to "Comics Panel Extractor", since western comics are kind of in between manga and manhwa in the sense of panel structures and being colored.

In short, this won't be fixed because
1. As of now I fail to do so
2. Even though I have a few different plans in mind for the project I have not decided on one particular of them and I can only make the decision of supporting other media types after being sure on this

I appreciate the effort of recording a video :+1:
